Shared/Samples/Create and save KML file/CreateAndSaveKMLView.Views.swift

```diff
@@ -0,0 +1,255 @@
+            } label: {
+                Label("Save Sketch", systemImage: "square.and.arrow.down")
+            }
+            .disabled(!model.canSave)
+            
+            Button {
+                model.stop()
+            } label: {
+                Label("Cancel Sketch", systemImage: "xmark")
+            }
+        }
+    }
+    
+    /// The point style menu.
+    var pointStyleMenuContent: some View {
+        VStack {
+            Button {
+                model.kmlStyle = KMLStyle(iconURL: URL(string: "http://resources.esri.com/help/900/arcgisexplorer/sdk/doc/bitmaps/148cca9a-87a8-42bd-9da4-5fe427b6fb7b127.png")!)
+                model.startGeometryEditor(withType: Point.self)
+            } label: {
+                Text("No Style")
+            }
+            
+            Button {
+                model.kmlStyle = KMLStyle(iconURL: URL(string: "https://static.arcgis.com/images/Symbols/Shapes/BlueStarLargeB.png")!)
+                model.startGeometryEditor(withType: Point.self)
+            } label: {
+                Text("Star")
+            }
+            
+            Button {
+                model.kmlStyle = KMLStyle(iconURL: URL(string: "https://static.arcgis.com/images/Symbols/Shapes/BlueDiamondLargeB.png")!)
+                model.startGeometryEditor(withType: Point.self)
+            } label: {
+                Text("Diamond")
+            }
+            
+            Button {
+                model.kmlStyle = KMLStyle(iconURL: URL(string: "https://static.arcgis.com/images/Symbols/Shapes/BlueCircleLargeB.png")!)
+                model.startGeometryEditor(withType: Point.self)
+            } label: {
+                Text("Circle")
+            }
+            
+            Button {
+                model.kmlStyle = KMLStyle(iconURL: URL(string: "https://static.arcgis.com/images/Symbols/Shapes/BlueSquareLargeB.png")!)
+                model.startGeometryEditor(withType: Point.self)
+            } label: {
+                Text("Square")
+            }
+            
+            Button {
+                model.kmlStyle = KMLStyle(iconURL: URL(string: "https://static.arcgis.com/images/Symbols/Shapes/BluePin1LargeB.png")!)
+                model.startGeometryEditor(withType: Point.self)
+            } label: {
+                Text("Round pin")
+            }
+            
+            Button {
+                model.kmlStyle = KMLStyle(iconURL: URL(string: "https://static.arcgis.com/images/Symbols/Shapes/BluePin2LargeB.png")!)
+                model.startGeometryEditor(withType: Point.self)
+            } label: {
+                Text("Square pin")
+            }
+        }
+    }
+    
+    /// The polyline style menu.
+    var polylineStyleMenuContent: some View {
+        VStack {
+            Button {
+                model.kmlStyle = KMLStyle(lineColor: .red)
+                model.startGeometryEditor(withType: Polyline.self)
+            } label: {
+                Text("Red")
+            }
+
```

Comment:
Inconsistent whitespace starting here and between the remaining buttons in this view and `polygonStyleMenuContent` below.

```suggestion
            
```